db/migrate/20180613155411_change_feed_source_site_url_to_hostname.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeFeedSourceSiteUrlToHostname < ActiveRecord::Migration[5.1]
```

Comment:
@MythicalFish Could you please provide your thought process for changing the schema?

This will require propagating a lot of changes on the data engineering side, so I'd strongly prefer to avoid this situation if at all possible!

Reply:
Fixed the migration as discussed :)